docs/root/operations/admin.rst

```diff
@@ -558,3 +558,7 @@ modify different aspects of the server:
   been configured to accept admin configuration. See:
 
   * :ref:`HTTP tap filter configuration <config_http_filters_tap_admin_handler>`
+
+.. http:get:: /reopen_logs
```

Comment:
Sorry that I didn't get this before, but I think this should be a POST as it changes state. cc @jmarantz

Reply:
np, will update today.

Reply:
agreed.